admin_guide/diagnostics_tool.adoc

```diff
@@ -378,8 +373,8 @@ ifdef::openshift-origin[]
 endif::[]
 image via the Docker CLI.
 
-As a non-root user that has privileges to run containers specify the cluster's
-inventory file and the *_health.yml_* playbook:
+Run the following as a non-root user that has privileges to run containers and
+specify the cluster's inventory file and the *_health.yml_* playbook:
```

Comment:
Do we need to say "specify the cluster's inventory file and the *_health.yml_* playbook" here? Would adding an annotation to line 383 be better? If we do, I'd s/specify/specify both